### PR TITLE
Add extensions for Completable

### PR DESCRIPTION
**Pull request by pt2121**
Not sure if `@Beta` and `@Experimental` stuff should be added here.

### CODE REVIEW

File: src/test/kotlin/rx/lang/kotlin/CompletableTest.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @test fun testCreateFromAction() {
+        var count = 0
+        val c1 = Action0 { count++ }.toCompletable()
+        assertNotNull(c1)
+        c1.subscribe()
+        assertEquals(1, count)
+
+        count = 0
+        val c2 = { count++ }.toCompletable()
```
Comment (cy6erGn0m) [inline src/test/kotlin/rx/lang/kotlin/CompletableTest.kt]: wouldn't it look better `completableOf { }` instead of `{}.toCompletable()` ?

**Comment by prt2121** [comment]
OK. I will update it.

File: src/test/kotlin/rx/lang/kotlin/CompletableTest.kt (hunk 1 above)
Comment (pt2121) [inline src/test/kotlin/rx/lang/kotlin/CompletableTest.kt]: OK. I will update it.

**Comment by cy6erGn0m** [comment]
lgtm, @MarioAriasC ?

**Comment by MarioAriasC** [comment]
:+1:

**Comment by pt2121** [comment]
Thanks!

**Pull request merged**